docs/test/test-guide.adoc

```diff
@@ -0,0 +1,30 @@
+= Test Guide
+:toc: macro
+:toc-title:
+:toclevels: 1
+
+toc::[]
+
+== Setting up test environment
+
+Requires *Go 1.14*
+
+Testing happens with the above version. Developers are advised to stick to this version.
+
+Unit test does not require any precondition, run `make test` to validate unit tests.   
+
+== Tests
+
+*Prerequisites for OpenShift cluster:*
+
+* A `crc` environment for 4.* local cluster:
```

Comment:
What is `4.*`?   It does not seem to markup the text.

Reply:
@wtam2018 `4.*` represents supported openshift cluster version, for example 4.1, 4.2 etc. We are running test on 4.5 in CI and for manual validation developers are using 4.6. So can i change it something like ```4.5+```.

Reply:
Done